example_caller/__tests__/test_adc.test.ts

```diff
@@ -0,0 +1,91 @@
+import { Principal } from "@dfinity/principal";
+import { beforeAll, describe, expect, test, Vitest, vitest } from "vitest";
+import { getCanisterCycles, ADC_CALLER, PROCESSOR_CALLER,processor_canister, adc_caller } from "./actor";
+import axios from "axios"
+
+
+
+
+let verifier:string;
+
+const target_url="https://api-testnet.nearblocks.io/v1/account/x-bitte-nfts.testnet/txns-only?cursor=0&order=asc";
+const method="GET"
+const redacted=""
+const headers=[]
+const body=""
+
+
+function wait(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe("ADC-caller IC Verifier", () => {
+
+  beforeAll(async()=>{
+    const notary_info = await axios.get(`${process.env.NOTARY_URL??"http://127.0.0.1:8080"}/notaryinfo`)
+    expect(notary_info.status,"Notary is no active").toBe(200)
+    expect(notary_info.data.publicKey.startsWith("-----BEGIN PUBLIC KEY-----")).toBe(true)
```

Comment:
notary_info is requested from the prover, so the env variable have to be `PROVER_URL`